core/src/main/scala/org/genivi/sota/core/data/Vehicle.scala

```diff
@@ -16,4 +17,13 @@ object Vehicle extends VehicleCommon {
 
   val MissingVehicle = new ErrorCode("missing_vehicle")
 
+  val genVehicle: Gen[Vehicle] =
```

Comment:
Not as it stands, but in the future when the notion of `Vehicle` will be different between core and resolver there will have to be some duplication -- so I set things up anticipating that.